R/parse.r

```diff
@@ -12,107 +12,85 @@ parse_all <- function(x) UseMethod("parse_all")
 
 #' @export
 parse_all.character <- function(x) {
-  x <- unlist(str_split(x, "\n"), recursive = FALSE, use.names = FALSE)
+
+  op <- options(stringsAsFactors = FALSE)
```

Comment:
This saves some typing, and is safe because you have `on.exit` but it makes the code harder to read because you need to understand more about the sequence of operations.